packages/sitecore-jss-angular/src/public_api.ts

```diff
@@ -80,10 +81,20 @@ export {
   enableDebug,
   ClientError,
   HTMLLink,
+  debug,
   CacheClient,
   CacheOptions,
   MemoryCacheClient,
 } from '@sitecore-jss/sitecore-jss';
+export {
+  GraphQLPersonalizeService,
+  GraphQLPersonalizeServiceConfig,
+  PersonalizeInfo,
+  CdpHelper,
```

Comment:
the import of CdpHelper at the end of this file should be removed to fix the build